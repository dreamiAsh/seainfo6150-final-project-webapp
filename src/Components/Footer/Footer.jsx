import React from "react";
import styles from "./Footer.module.css";
import { Link, BrowserRouter } from "react-router-dom";

const Footer = (props) => {
  return (
    <div className={styles.box}>
        <div className={styles.leftBox}>
            <BrowserRouter forceRefresh={true}><Link to="/aboutUs" className={styles.item}>About us</Link></BrowserRouter>
        </div>
        <div className={styles.middleBox}>
            <BrowserRouter forceRefresh={true}><Link to="/termService" className={styles.item}>Terms of Service</Link></BrowserRouter>
            <p className={styles.itemCompany}>Â© Copyright 2020 Home recipe Corporation</p>
        </div>
        <div className={styles.rightBox}>
            <BrowserRouter forceRefresh={true}><Link to="/privacy" className={styles.item}>Privacy policy</Link></BrowserRouter>
        </div>
    </div>
  );
};

export default Footer;
