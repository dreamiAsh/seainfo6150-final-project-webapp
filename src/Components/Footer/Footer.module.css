.box{
    width: 100%;
    height: 90px;
    background: #DC7E20;
    display: grid;
    flex-flow: column-reverse;
    justify-content: space-around;
    align-items: stretch;
    grid-template-columns: repeat(3, 30fr);
    text-align: center;
    color: white;
    padding-top: 40px;
}
.item{
    margin-top: 10px;
    text-decoration: none;
    color: white;
    display: block;
}
.item:hover{
    text-decoration: underline;
}
.itemCompany{
    margin-top: 40px;
    font-size: 13px;
}