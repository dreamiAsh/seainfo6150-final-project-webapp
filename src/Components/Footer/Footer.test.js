import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Footer from "./Footer.jsx";

it("renders the footer component correctly", () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
});