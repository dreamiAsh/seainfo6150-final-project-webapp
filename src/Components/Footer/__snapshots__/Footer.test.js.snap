// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the footer component correctly 1`] = `
<div>
  <div
    class="box"
  >
    <div
      class="leftBox"
    >
      <a
        class="item"
        href="/aboutUs"
      >
        About us
      </a>
    </div>
    <div
      class="middleBox"
    >
      <a
        class="item"
        href="/termService"
      >
        Terms of Service
      </a>
      <p
        class="itemCompany"
      >
        Â© Copyright 2020 Home recipe Corporation
      </p>
    </div>
    <div
      class="rightBox"
    >
      <a
        class="item"
        href="/privacy"
      >
        Privacy policy
      </a>
    </div>
  </div>
</div>
`;
